README.md

```diff
@@ -80,6 +80,17 @@ As an optimization, storage systems can also allow _partial listing of a directo
 
 Delta Lake ensures _serializability_ for concurrent reads and writes. Please see [Delta Lake Concurrency Control](https://docs.delta.io/latest/delta-concurrency.html) for more details.
 
+# Managed Delta Lake on Databricks Features Not Supported
```

Comment:
Thanks for working on this issue! I agree there has been some confusion on our roadmap and it would be good to address it.

That said, some of the things listed here are deeply tied to the specifics of Databricks runtime and likely are not candidates to be added to the Delta Lake project. Given that, I don't think it makes sense to "plug" databricks features here. I think it makes more sense for a list of databricks-only features to live on databricks.com.

Instead, I think we can add a more generic statement about what we plan to open source and provide a link to the detailed roadmap, as you did. I've asked the team to create issues for all the things that are in progress so that the milestones link is more complete.

I'd probably put text closer to the top under `# Latest Binaries`.  What do you think of the following?

# Roadmap
Delta Lake is a recent open source project, based on technology developed at [Databricks](https://databricks.com/delta/). We plan to open-source all APIs that are required to correctly run Spark programs that read and write Delta tables. For a detailed timeline on this effort see the project [roadmap](https://github.com/delta-io/delta/milestones).

Reply:
Yeah I like that. Honestly, you might not even want to overload the README with too much information (whether that includes the roadmap or not). Some of this stuff would be good on the delta.io page or under the FAQ. In fact your note on databricks-only features would be an excellent FAQ item. For example a general question like, "Why is a feature I see in the databricks documentation not included in the OSS project?"

I'll add the Roadmap section towards the top of my PR changes.